app/services/FileService.scala

```diff
@@ -74,7 +74,8 @@ class FileService @Inject()() extends Logging {
 
   private def createManifest(files: List[FileAsBytes]): Option[FileAsBytes] = {
     Try {
-      val map: Map[String, String] = files.map { p => (p.filename, Hashing.sha1().hashBytes(p.content).toString) }.toMap
+      // TODO is it okay to replace sha1() with sha256()?
```

Comment:
I would potentially be cautious with this one. Might be better to keep that as it is? Maybe reach out and ask Joe S or Pascal?